internal/migrations/migrate_collection.go

```diff
@@ -13,6 +14,17 @@ func MigrateCollectionSpec(spec logging.ClusterLoggingSpec) logging.ClusterLoggi
 		return spec
 	}
 
+	if spec.Visualization == nil && spec.LogStore != nil && spec.LogStore.Type == logging.LogStoreTypeLokiStack {
```

Comment:
Please add this to its own migration file (e.g. migrate_visualization.go)

Reply:
done